podcasts/ExpandableLabel.swift

```diff
@@ -32,7 +32,36 @@ class ExpandableLabel: ThemeableLabel {
         collapsed = linesRequired() > maxLines
     }
 
-    @objc private func labelTapped() {
+    func setRichText(html: String) {
+        let styledHTML: String = """
+        <html>
+        <head>
+        <style>
+        body {
+            font-family: -apple-system;
+            font-size: 1.5em;
+        }
+        </style>
+        </head>
+        <body>
+        \(html)
+        </body>
+        </html>
+        """
+        guard let data = styledHTML.data(using: .utf8) else {
+            return
+        }
+        let result = try? NSAttributedString(data: data, options: [.documentType: NSAttributedString.DocumentType.html, .characterEncoding: NSUTF8StringEncoding], documentAttributes: nil)
+        attributedText = result
+        textColor = AppTheme.colorForStyle(style)
+        collapsed = linesRequired() > maxLines
+    }
+
+    @objc private func labelTapped(gesture: UITapGestureRecognizer) {
+        if let url = gesture.didTapLinkInLabel(label: self) {
+            UIApplication.shared.open(url)
```

Comment:
Should we open this externally or internally?

Reply:
I followed the same pattern that we have on the page. If you scroll a bit down you will see a section with the podcast url and if you tap on it you are taken to the external browser